hs_core/views/utils.py

```diff
@@ -82,11 +81,11 @@ def upload_from_irods(username, password, host, port, zone, irods_fnames, res_fi
     ifnames = string.split(irods_fnames, ',')
     for ifname in ifnames:
         size = irods_storage.size(ifname)
-        if size > FILE_SIZE_LIMIT:
-            raise_file_size_exception()
         tmpFile = irods_storage.download(ifname)
```

Comment:
@pkdash For very big files, celery task is probably better, but since it does not involve web client, but only irods parallel file transfer, the wait time is really reasonable based on my testing, According to my testing on playground, a file size of 2GB only takes a few seconds to create a resource out of it, so I assume the file transfer for gigabyte files at this time is probably acceptable without resorting to celery task. However, if a need arises in the future, we can definitely consider moving this into a celery task. Since a celery task implementation is more involved due to the need for notification and front-end polling, I think we can go with this for now unless there is a need to do so prompted by a real use case.